cli/src/semgrep/metrics.py

```diff
@@ -297,6 +303,17 @@ def add_targets(self, targets: Set[Path], profiling_data: ProfilingData) -> None
         self.payload["performance"]["totalBytesScanned"] = total_bytes_scanned
         self.payload["performance"]["numTargets"] = len(targets)
 
+    @suppress_errors
+    def add_ignores(self, ignored: Set[Path]) -> None:
+        ignored_ext_freqs: Dict[str, int] = {}
```

Comment:
might wanna use https://docs.python.org/3/library/collections.html#collections.Counter for simpler code

Reply:
So much simpler now! Thanks :)